bindings/rust/s2n-tls/src/raw/connection.rs

```diff
@@ -172,10 +115,10 @@ impl Connection {
     /// S2N_CERT_AUTH_OPTIONAL will terminate the handshake.
     pub fn set_client_auth_type(
```

Comment:
can you update the docs above to the new enum name.

S2N_CERT_AUTH_OPTIONAL -> ClientAuthType::Optional